theme_kea/views/snippets/s_media_list.xml

```diff
@@ -3,51 +3,67 @@
     <template id="s_media_list" inherit_id="website.s_media_list">
         <!-- Section -->
         <xpath expr="//section" position="attributes">
-            <attribute name="class" add="o_cc4" remove="o_cc2" separator=" "/>
+            <attribute name="class" add="o_cc2 pt64 pb64" remove="o_cc1 pt32 pb32" separator=" "/>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Floats/01","flip":[]}</attribute>
+        </xpath>
+        <!-- Shapes -->
+        <xpath expr="//div[hasclass('container')]" position="before">
+            <div class="o_we_shape o_web_editor_Floats_01"/>
         </xpath>
         <!-- Row 3 -->
         <xpath expr="(//div[hasclass('row')])[4]" position="attributes">
-            <attribute name="class" add="o_cc5 flex-row-reverse rounded" remove="o_cc1" separator=" "/>
-            <attribute name="style" add="border-radius: 25px !important;" separator=" "/>
+            <attribute name="class" remove="o_cc1" separator=" "/>
         </xpath>
         <xpath expr="(//div[hasclass('col-lg-4')])[3]" position="attributes">
-            <attribute name="class" add="col-lg-3" remove="col-lg-4" separator=" "/>
+            <attribute name="class" add="col-lg-6" remove="col-lg-4" separator=" "/>
+        </xpath>
+        <xpath expr="(//img)[3]" position="attributes">
+            <attribute name="style" add="border-radius:200px !important" separator=" "/>
```

Comment:
This should be an option available in the editor with other image style options (rounded, square, thumbnail...).
In BS, there is a class and a variable that we could use:
```scss
$rounded-pill: 50rem !default;

.rounded-pill {
    border-radius: $rounded-pill !important;
}